public_website/__tests__/pages/index.test.tsx

```diff
@@ -14,14 +14,18 @@ describe('Home page', () => {
     }
   })
 
-  it('should pass accessibility tests', async () => {
-    const { props } = await getStaticProps()
-    const { container } = render(<Home {...props} />)
+  it(
+    'should pass accessibility tests',
+    async () => {
```

Comment:
L'indentation me semble bizarre, c'était mieux avant
```suggestion
  it('should pass accessibility tests', async () => {
```

Reply:
J'avoue que c'est chelou mais avec le 3eme argument sur le test, ESLint le met comme ça par défaut 🤨